x/incentives/keeper/distribute_test.go

```diff
@@ -726,6 +746,14 @@ func (suite *KeeperTestSuite) TestDistributeConcentratedLiquidity() {
 		suite.Run(tc.name, func() {
 			suite.SetupTest()
 			suite.Ctx = suite.Ctx.WithBlockTime(defaultBlockTime)
+
+			// Set up authorized CL uptimes to robustly test distribution
+			if tc.authorizedUptimes != nil {
+				clParams := suite.App.ConcentratedLiquidityKeeper.GetParams(suite.Ctx)
+				clParams.AuthorizedUptimes = tc.authorizedUptimes
+				suite.App.ConcentratedLiquidityKeeper.SetParams(suite.Ctx, clParams)
+			}
```

Comment:
Same comment here

Reply:
See c31100c75598a516e5b3106140f3b4c0654549f5 and ec471d648016de079af19f01b52771e6e5f98c39